tests/test_root_model.py

```diff
@@ -221,6 +232,145 @@ def double(cls, v):
             v.root *= 2
             return v
 
-    Model('1')
     assert Model('1').root == 2
     assert Model('21').root == 42
+
+
+def test_private_attr():
+    class Model(RootModel[int]):
+        _private_attr: str
+        _private_attr_default: str = PrivateAttr(default='abc')
+
+    m = Model(42)
+
+    assert m.root == 42
+    assert m._private_attr_default == 'abc'
+    with pytest.raises(AttributeError, match='_private_attr'):
+        m._private_attr
+
+    m._private_attr = 7
+    m._private_attr_default = 8
+    m._other_private_attr = 9
+    # TODO: Should this be an `AttributeError`?
+    with pytest.raises(ValueError, match='other_attr'):
+        m.other_attr = 10
```

Comment:
It shouldn't be `ValueError`, but not sure if it should be something different than `AttributeError`.

Reply:
Are you going to use `AttributeError` on this PR then?

Reply:
Not in this one. Maybe later. That is why I did put `TODO` here:)